book/src/web-client/create_deploy_tutorial.md

```diff
@@ -0,0 +1,261 @@
+
+ A wallet on Miden is simply an account with standardized code.
+
+In the example below we create a mutable public account for Alice. 
+
+Add this snippet to the `webClient()` function:
+```ts
+const aliceAccount = await client.new_wallet(
+  AccountStorageMode.public(), // account type
+  true                         // mutability
+);
+const aliceIdHex = aliceAccount.id().to_string();
+console.log("Alice's account ID:", aliceIdHex);
+
+await client.sync_state();
+```
+
+## Step 4: Deploying a fungible faucet
+For Alice to receive testnet assets, we first need to deploy a faucet. A faucet account on Miden mints fungible tokens.
+
+We'll create a public faucet with a token symbol, decimals, and a max supply. We will use this faucet to mint tokens to Alice's account in the next section.
+
+Add this snippet to the end of the `webClient()` function:
+```ts
+const faucetAccount = await client.new_faucet(
+  AccountStorageMode.public(), // account type
+  false,                       // is fungible
+  "MID",                       // symbol
+  8,                           // decimals
+  BigInt(1_000_000)            // max supply
+);
+const faucetIdHex = faucetAccount.id().to_string();
+console.log("Faucet account ID:", faucetIdHex);
+
+await client.sync_state();
+```
+
+*When tokens are minted from this faucet, each token batch is represented as a "note" (UTXO). You can think of a Miden Note as a cryptographic cashier's check that has certain spend conditions attached to it.*
+
+## Summary
+Our new `src/webClient.ts` file should look something like this:
+
+```ts
+import { WebClient, AccountStorageMode } from "@demox-labs/miden-sdk";
+
+const nodeEndpoint = "http://localhost:57291";
+
+export async function webClient(): Promise<void> {
+  try {
+    // 1. Create client
+    const client = new WebClient();
+    await client.create_client(nodeEndpoint);
+
+    // 2. Sync and log block
+    const state = await client.sync_state();
+    console.log("Latest block number:", state.block_num());
+
+    // 3. Create Alice account (public, updatable)
+    const aliceAccount = await client.new_wallet(
+      AccountStorageMode.public(),  // account ty[e]
+      true                          // mutability
+    );
+    const aliceIdHex = aliceAccount.id().to_string();
+    console.log("Alice's account ID:", aliceIdHex);
+
+    // 4. Create faucet
+    console.log("Creating faucet...");
+    const faucetAccount = await client.new_faucet(
+      AccountStorageMode.public(), // account type
+      false,                       // is fungible
+      "MID",                       // symbol
+      8,                           // decimals
+      BigInt(1_000_000)            // max supply
+    );
+    const faucetIdHex = faucetAccount.id().to_string();
+    console.log("Faucet account ID:", faucetIdHex);
+
+    await client.sync_state();
+  } catch (error) {
+    console.error("Error", error);
+    throw error;
+  }
+}
```

Comment:
<img width="542" alt="Screenshot 2025-01-23 at 10 15 38" src="https://github.com/user-attachments/assets/5b0b1945-e8bc-4828-add5-47db7705d5e0" />

I deleted the accounts DB in the node and ran it again. Now I get this output. 

Not sure how to handle that. But this is a bad DevEx. Have you seen that before?